pkg/informer/informer.go

```diff
@@ -179,7 +179,7 @@ func SupportedResources(discoveryClient *discovery.DiscoveryClient) (map[schema.
 
 	//locate the allow-deny ConfigMap:
 	var cm *v1.ConfigMap
-	if cm, err = clientset.CoreV1().ConfigMaps("POD_NAMESPACE").Get(contextVar, "allowdeny-config", metav1.GetOptions{}); err != nil {
+	if cm, _ = clientset.CoreV1().ConfigMaps("POD_NAMESPACE").Get(contextVar, "allowdeny-config", metav1.GetOptions{}); err != nil {
```

Comment:
Not sure why we are removing err here .

Reply:
this is old @xrajesh, updated commits have the error.

Reply:
everything new is moved into it's own file `supportedResources.go`